unlock-js/src/__tests__/integration/walletServiceIntegration.test.js

```diff
@@ -0,0 +1,46 @@
+import nock from 'nock'
+import WalletService from '../../walletService'
+
+describe('Wallet Service Integration', () => {
+  let walletService
+  let provider
+
+  beforeAll(async () => {
+    nock.enableNetConnect()
```

Comment:
don't use `beforeAll` or `beforeEach` with nock. This is for 2 reasons:

1. nock modifies global state, and so can affect other tests if not cleaned up
2. `beforeEach` and `beforeAll` always run, even when the tests inside are not focused

These 2 things combined can cause tests to pass and then fail when run focused as the setup for other tests will run for the focused test.